packages/design-system-scripts/cli.js

```diff
@@ -56,8 +64,13 @@ yargs
     },
     handler: async argv => {
       logIntroduction();
-      const { startDocsServer } = require('./gulp/server');
-      await startDocsServer(argv.sourcePackageDir, argv.docsPackageDir, { ...argv });
+      const { buildSrc } = require('./gulp/build');
```

Comment:
I reorganized these main task to be more granular so that we have better control of which tasks are run and in which order. I.e. if we break out `printStats` from `build` we can run it last in the case of `build-docs`. This was also helpful for `watch` task, which needed to run specific parts of each task depending on the file type it is watching

Reply:
💯

Reply:
I figured I'd have to refactor these as I went along but just wanted a working basic build script at first